Mathlib/LinearAlgebra/AffineSpace/AffineSubspace.lean

```diff
@@ -346,9 +346,9 @@ theorem ext {p q : AffineSubspace k P} (h : ∀ x, x ∈ p ↔ x ∈ q) : p = q
 #align affine_subspace.ext AffineSubspace.ext
 
 -- Porting note: removed `simp`, proof is `simp only [SetLike.ext'_iff]`
-theorem ext_iff (s₁ s₂ : AffineSubspace k P) : (s₁ : Set P) = s₂ ↔ s₁ = s₂ :=
+theorem ext_iff' (s₁ s₂ : AffineSubspace k P) : (s₁ : Set P) = s₂ ↔ s₁ = s₂ :=
   SetLike.ext'_iff.symm
```

Comment:
Another one

Reply:
As above, not sure what change on master you had in mind.